apps/api/src/domain/games/memory-match/queries/memory-match-queries.fetch-many.e2e.spec.ts

```diff
@@ -122,4 +157,86 @@ describe(`when querying for a memory match round: fetch many`, () => {
             });
         });
     });
+
+    describe(`when the user is authenticated as a viewer`, () => {
+        beforeAll(async () => {
+            await setItUp(new CoscradUserWithGroups(nonAdminUser, []));
+        });
+
+        describe(`when no filters are provided`, () => {
+            it(`should return all published rounds`, async () => {
+                const res = await request(app.getHttpServer()).get(indexEndpoint);
+
+                expect(res.status).toBe(HttpStatusCode.ok);
+
+                const {
+                    body: { entities },
+                } = res;
+
+                // there should be two rounds visible to authenticated user
```

Comment:
This comment is wrong. 
there should be one round visible to an ordinary viewer (non-admin)